lib/features/modeling/behavior/EventBasedGatewayBehavior.js

```diff
@@ -0,0 +1,73 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { is } from '../../../util/ModelUtil';
+
+export default function EventBasedGatewayBehavior(eventBus, modeling) {
+
+  CommandInterceptor.call(this, eventBus);
+
+  /**
+   * Remove existing sequence flows of event-based target before connecting
+   * from event-based gateway.
+   */
+  this.preExecuted('connection.create', function(event) {
+
+    var source = event.context.source,
+        target = event.context.target,
+        existingIncomingConnections = target.incoming.slice();
+
+    if (
+      is(source, 'bpmn:EventBasedGateway') &&
+      target.incoming.length > 0
+    ) {
+
+      existingIncomingConnections
+        .filter(function(connection) {
+          return is(connection, 'bpmn:SequenceFlow');
```

Comment:
We check this condition multiple times in the file so why don't we have a helper for this? It will increase the readibility of the code :)

Reply:
```suggestion
filter(isSequenceFlow)
```